docs/doc-structure.md

```diff
@@ -0,0 +1,99 @@
+---
+id: doc-structure
+title: Integration Doc Format
+---
+
+## Overview
+Type of the product and common use case through Demisto.
+  What is this integration good for? 
+  What does the integration do? 
+Known limitations (only if needed in this high level view)
+What version of the integrated product was tested (and what versions we believe are supported. e.g. tested on 2.0, should work on 2.0 and up) 
+
+## To set up [integration name] to work with Demisto:
+Just list the requirements for integrating with Demisto. You can include links to third-party documentation if necessary.
+
+* API token
+* Credentials 
+* Etc
+
+## To set up the integration on Demisto:
+For example:
+1. Go to ‘Settings > Integrations > Servers & Services’
```

Comment:
Steps not rendered as a list. Suggest additional new line after "For example: ".

Reply:
will fix